database/init.go

```diff
@@ -93,8 +93,8 @@ func createConstantTable(tableName string) (err error) {
 	query :=
 		fmt.Sprintf("CREATE TABLE IF NOT EXISTS %s (", tableName) +
 			`name character varying(20) NOT NULL PRIMARY KEY,
-			created_at timestamp with time zone DEFAULT CURRENT_TIMESTAMP,
-			updated_at timestamp with time zone DEFAULT CURRENT_TIMESTAMP,
+			created_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
+			updated_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
 			deleted_at timestamp with time zone);`
```

Comment:
What is this for? If we delete, the record will be lost. Or there is other column to mark it as deleted

Reply:
If you use the endpoint `delete`, the `deleted_at` column will be set instead of literally delete the record.